### ISSUE TITLE
Quality assuance and testing

### ISSUE THREAD

**Issue by Troy-Lawson** [gctools-outilsgc/profile_service_react#25]
product owner signoff
UX testing
peer code review

**Comment by MPM9000-zz** [comment]
For style references as editable PDF:
1) designs for wizard state modal (for onboarding)
[US01_Directory_New_User_Onboarding_V02.pdf](https://zube.io/files/tbs-sct/401e876ed2f2375c443faea85a15598c-us01_directory_new_user_onboarding_v02.pdf)

2) design for editing a profile:
[US_01b_Edit_Profile_v02 2.pdf](https://zube.io/files/tbs-sct/79a8b3265a37321794dad502dde6c745-us_01b_edit_profile_v02-2.pdf)

optional:
3) design for supervisor's teams (if we get that far)
[US12_Supervisor_Creates_Team_v03.pdf](https://zube.io/files/tbs-sct/80da45c927797f6b1802c39bda8d669f-us12_supervisor_creates_team_v03.pdf)

@sabrinatbs @piet0024

**Comment by sabrinatbs** [comment]
@piet0024 

Bug link for onboarding: http://localhost:3000/onboard

(fixed)

**Comment by sabrinatbs** [comment]
Bug when trying to add supervisor: 

https://github.com/piet0024/directory-fe/issues/3

![image.png](https://zube.io/files/tbs-sct/798a6d16405b0df3845a9f77f35e752c-image.png)

![image.png](https://zube.io/files/tbs-sct/a1e9c8a66beee703ebb25f0e3ff34d16-image.png)

**Comment by sabrinatbs** [comment]
@piet0024 I was able to complete the assessment. The first file you will find is a document that lists the needed modifications. The second document is a scanned copy of Mike's designs that indicate the changes. The first document also lists the bugs we encountered and some UX testing that should be revisited once the changes are complete. I printed both of these copies and but them on your desk. Let me know if you have any questions or need clarifications.

[Directory Quality Assurance.pdf](https://zube.io/files/tbs-sct/77df0d3bc2427b4fa4a130fad1340e19-directory-quality-assurance.pdf)

[Directory_Modifications.pdf](https://zube.io/files/tbs-sct/cce341fd7796b25431fabef73a353aa1-directory_modifications.pdf)
